pyproject.toml

```
@@ -1,5 +1,5 @@
 [build-system]
-requires = ["hatchling>=1.5", "jupyterlab>=4.0.0a25,<5", "ypy-websocket==0.2"]
+requires = ["hatchling>=1.5", "jupyterlab>=4.0.0a29,<5", "ypy-websocket==0.3.2"]
```

It should be possible to drop that one as well:

```suggestion
requires = ["hatchling>=1.5", "jupyterlab>=4.0.0a29,<5"]
```